### PR TITLE
virtuoso: update to 7.2.10

### PR DESCRIPTION
**Pull request by klarasm**
#### Testing the changes
- I tested the changes in this PR: most of the test suite passes (excepting tgraphql), I don't use this package directly

This package has some weirdness in the test suite. It fails when run from `./xbps-src -K pkg virtuoso` but succeeds if I do `./xbps-src chroot` afterwards and:
```
make clean
./autogen.sh
./configure
make
make check
```

I tried replacing pre_configure, do_configure, do_build and do_check to do the same thing but it still fails. Maybe I'm missing some dependencies or something.

`tgraphql` is the only test failing: 
[tgraphql.output.log](https://github.com/void-linux/void-packages/files/10926435/tgraphql.output.log)

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for my native architecture, (x86_64-musl)

### CODE REVIEW

**Comment by klarasm** [comment]
I disabled tgraphql test for now. As I don't use this package directly I don't know if the resulting package works correctly.

**Comment by klarasm** [comment]
Seems like thttp is failing on musl as well but the error does not appear on my x86_64-musl machine.

**Comment by klarasm** [comment]
The thttp failure does not seem to be consistent. If anyone that uses virtuoso could test the changes I would appreciate it.

**Comment by klarasm** [comment]
Bump. If there's any active user of this package that can test, I would appreciate it.

File: srcpkgs/virtuoso/template [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-hostmakedepends="automake libtool flex gperf net-tools"
+configure_args="--without-internal-zlib --with-readline --with-layout=debian"
+hostmakedepends="automake libtool flex gperf net-tools which"
 makedepends="openssl-devel libldap-devel readline-devel libxml2-devel mit-krb5-devel"
+checkdepends="procps-ng unzip curl wget"
 short_desc="Scalable cross-platform server of virtuoso"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2"
+license="GPL-2.0-only"
 homepage="http://virtuoso.openlinksw.com/wiki/main/Main/"
```
Comment (Duncaen) [inline srcpkgs/virtuoso/template:14]: Redirects to https://vos.openlinksw.com/owiki/wiki/VOS/

Comment (klarasm) [reply srcpkgs/virtuoso/template]: Updated to the new one. There's also https://virtuoso.openlinksw.com/ but that's not for the open source version.

**Pull request merged**